nexus_constructor/model/load_from_json.py

```diff
@@ -53,9 +78,217 @@ def _retrieve_children_list(json_dict: dict) -> list:
+                except KeyError:
+                    continue
+
+    def _get_transformation_property(
+        self,
+        property_name: str,
+        json_transformation: dict,
+        transform_name: str = None,
+        failure_value: Any = None,
+        parse_result_func: Callable = None,
+    ) -> Any:
+        """
+        Tries to find a certain property of a transformation from dictionary.
+        :param property_name: The name of the property fields.
+        :param json_transformation: The dictionary to look for the property in.
+        :param transform_name: The name of the transformation (if known).
+        :param failure_value: The value to return if the property cannot be found.
+        :param parse_result_func: A function that should be called on the dictionary value if it is found.
+        :return: Returns the property or converted property if this exists in the dictionary, if the property is not
+        found in the dictionary then the failure_value is returned.
+        """
+        try:
+            if not parse_result_func:
+                return json_transformation[property_name]
+            else:
+                return parse_result_func(json_transformation[property_name])
+        except KeyError:
+            if transform_name:
+                msg = (
+                    f"Cannot find {property_name} for transformation in component"
+                    f" {transform_name}"
+                )
+                f" {self.parent_component.name}."
+            else:
+                msg = f"Cannot find {property_name} for transformation in component"
+                f" {self.parent_component.name}."
+            self.warnings.append(msg)
+            return failure_value
+
+    def _find_property_in_list(
+        self,
+        property_name: str,
+        transformation_name: str,
+        attributes_list: list,
+        failure_value: Any = None,
+        parse_result_func: Callable = None,
+    ) -> Any:
+        """
+        Searches the dictionaries in a list to see if one of them has a given property.
+        :param property_name: The name of the property that is being looked for.
+        :param transformation_name: The name of the transformation that is being constructed.
+        :param attributes_list: The list of dictionaries.
+        :param failure_value: The value to return if the property is not contained in any of the dictionaries.
+        :param parse_result_func: todo: description
+        :return: The value of the property if is is found in the list, otherwise the failure value is returned.
+        """
+        for attribute in attributes_list:
+            try:
+                if attribute["name"] == property_name:
+                    if parse_result_func:
+                        return parse_result_func(attribute["values"])
+                    else:
+                        return attribute["values"]
+            except KeyError:
+                continue
```

Comment:
If I follow correctly, if you reach this line then you've found the property with the specified name but it had no "values" field. I don't think it makes sense to continue and try to look for another property with the same name:
`break` rather than `continue`?